modules/transport-netty4/src/internalClusterTest/java/org/elasticsearch/transport/netty4/ESLoggingHandlerIT.java

```diff
@@ -54,7 +52,7 @@ public void testLoggingHandler() {
             + ", request id: \\d+"
             + ", type: request"
             + ", version: .*"
-            + ", action: cluster:monitor/nodes/hot_threads\\[n\\]\\]"
+            + ", action: cluster:monitor/nodes/stats\\[n\\]\\]"
```

Comment:
Why do we need to change the action we use here? :) How come hot threads isn't good any more?

Reply:
It seems a little daft to use an action which, by design, takes over a second to respond. Not that these tests are especially efficient anyway but still.